### DISCUSSION TITLE
Seaching and Sorting

### DISCUSSION THREAD

**Discussion by salomaojunior** [dailydotdev/daily#789] [Feature Requests]
Hi guys, is it possible to add the sorting option in the search box? I would like to see the newest post in the search, but i couldnt find it.

**Comment by idoshamun** [comment]
Hi @salomaojunior,

Thanks a lot for the suggestion. We have an active discussion about the topic here: https://github.com/dailydotdev/daily/discussions/294.
Feel free to join the discussion
